src/components/views/Header.js

```diff
@@ -93,6 +101,25 @@ function Header(props){
             </div>
         }
     }
+    if(props.view==="userPage"){
+        return <div className="header container" style={{height:"100"}}>
```

Comment:
I would say it's better to delegate the height property completely to the CSS file, since it is always "100".

Reply:
Ahh good point. Wanted to do long ago but forgot. Done now